Source/Scene/PointCloud.js

```diff
@@ -456,7 +456,10 @@ function createResources(pointCloud, frameState) {
 
   let batchIdsVertexBuffer;
   if (hasBatchIds) {
-    batchIds = prepareVertexAttribute(batchIds, "batchIds");
+    batchIds.typedArray = prepareVertexAttribute(
+      batchIds.typedArray,
+      "batchIds"
+    );
```

Comment:
Caught this bug in the process of adding a DeveloperError to `ComponentDatatype`.